src/OrbitGl/App.cpp

```diff
@@ -348,71 +349,73 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
     metrics_capture_complete_data_.file_path = file_path.value();
   }
 
-  main_thread_executor_->Schedule(
-      [this, &initialization_complete, &mutex, &capture_started, file_path = std::move(file_path),
-       frame_track_function_ids = std::move(frame_track_function_ids)]() mutable {
-        absl::flat_hash_map<Track::Type, bool> track_type_visibility;
-        bool had_capture = capture_window_->GetTimeGraph();
-        if (had_capture) {
-          track_type_visibility =
-              capture_window_->GetTimeGraph()->GetTrackManager()->GetAllTrackTypesVisibility();
-        }
+  main_thread_executor_->Schedule([this, &initialization_complete, &mutex, &capture_started,
+                                   file_path = std::move(file_path),
+                                   frame_track_function_ids =
+                                       std::move(frame_track_function_ids)]() mutable {
+    api_event_id_provider_ =
+        orbit_gl::NameEqualityApiEventIdProvider::Create(capture_started.capture_options());
```

Comment:
This is the only new part here and everything else is indentation, right?

Reply:
Yep.

Reply:
Arg... Formatting in lambdas is still weird. Maybe we should move the code into a function at some point (not in this PR). It is quite long anyways.